xml/net_yast.xml

```diff
@@ -783,7 +783,7 @@
      <para>
       In <guimenu>Modify DNS Configuration</guimenu>, select the way the DNS
       configuration (name servers, search list, the content of the
-      <filename>/etc/resolv.conf</filename> file) is modified.
+      <filename>/run/netconfig/resolv.conf</filename> file) is modified.
```

Comment:
Not related to your changes, but this sentence is hard to read due to the text in the parenthesis. The enumeration "name servers, search list, the content of the /etc/resolv.conf file" doesn't make sense anyway. The first too are the settings that are modified while the latter is the file to store them. I think we should just remove the file name here to improve readability.

Reply:
Thanks, changed now.